python/src/robyn/visualization/base_visualizer.py

```diff
@@ -14,6 +14,7 @@
 
 # Configure logger
 logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
```

Comment:
Use should be able to control this. Setting log level should be at one time config level only.